src/praisonai/praisonai/cli/interactive/__init__.py

```
@@ -1,10 +1,45 @@
 """
 
-from praisonai.cli._shim import alias_package as _alias_package
+try:  # pragma: no cover - defensive
+    import praisonai_code.cli.interactive as _code_interactive
 
-_alias_package(__name__, "praisonai_code.cli.interactive")
+    for _code_dir in getattr(_code_interactive, "__path__", []):
+        if _code_dir not in __path__:
+            __path__.append(_code_dir)
+    del _code_dir
+except Exception:  # pragma: no cover - code package optional at import time
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)